README.md

```diff
@@ -1,2 +1,290 @@
-# Project-Hello
-A project to master tools and best practices for future projects
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
```

Comment:
- [ ] [OPTIONAL] It would be great if you could `delete` the unnecessary comments 👍🏼